admin_guide/backup_restore.adoc

```diff
@@ -133,8 +139,24 @@ inside the container.
 . Copy the *_db_* file over to the backup you created:
 +
 ----
-$ cp "$ETCD_DATA_DIR"/member/snap/db "$HOTDIR"/member/snap/db'
+# cp "$ETCD_DATA_DIR"/member/snap/db "$HOTDIR"/member/snap/db'
+----
+
+[[registry-certificates-backup]]
+=== Registry Certificates Backup
+
+. Save all the registry certificates, on every master and node host.
++
 ----
+# cd /etc/docker/certs.d/
+# tar cf /tmp/docker-registry-certs-$(hostname).tar * 
+----
++
+[NOTE]
+====
+When working with one or more xref:../install_config/registry/securing_and_exposing_registry#exposing-the-registry[external secured registry], 
```

Comment:
@jtudelag Oops! Yes,  you are correct. We just need to make sure that the path includes .adoc before the #exposing-the-registry.